Makefile.PL

```diff
@@ -4,7 +4,7 @@ name 'WebService-Prowl';
 all_from 'lib/WebService/Prowl.pm';
 
 requires 'LWP::UserAgent';
```

Comment:
should we change LWP::UserAgent version too?

`requires 'LWP::UserAgent' => '6.02';
`